java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCSToJDBC.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.dataproc.templates.gcs;
+
+import com.google.cloud.dataproc.dialects.SpannerJdbcDialect;
+import com.google.cloud.dataproc.templates.BaseTemplate;
+import com.google.cloud.dataproc.templates.util.PropertyUtil;
+import com.google.cloud.dataproc.templates.util.ValidationUtil;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.execution.datasources.jdbc.JDBCOptions;
+import org.apache.spark.sql.jdbc.JdbcDialects;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GCSToJDBC implements BaseTemplate {
+  public static final String JDBC_DRIVER = "com.mysql.jdbc.Driver";
```

Comment:
Having the JDBC driver configured this way makes the template only work for MySQL databases. Something like this works for the `GcsToSpanner` template because it only targets Spanner, but here we want the template to work with many different databases.

Like I mentioned in my pom comment, we can refer to the existing `JDBCTo*` templates for this. Those parametrize the driver class and require the user to add the relevant JARs at job submission.

Reply:
Removed that line, and now taking driver name via command line arguments. Thanks for the suggestion.